content/documentation/staging/faq/general/prom-metrics.adoc

```diff
@@ -0,0 +1,135 @@
+---
+title: Which Istio metrics and attributes are required by Kiali?
+weight: 70
+---
+
+To reduce Prometheus storage some users want to customize the metrics generated by Istio.  This can break Kiali if the pruned metrics and/or attributes are used by Kiali in its graph or metric features.
+
+Kiali currently requires the following metrics and attributes (note, this assumes Telemetry V2 in being used):
```

Comment:
perhaps having all the metrics/labels in a backquoted block would be easier to read ? (less scrolling, and mono fonts better for this kind of data)